### PR TITLE
Support gluster peer fqdn/ip

### PR DESCRIPTION
**Pull request by r0h4n**
tendrl-bug-id: Tendrl/commons#852

### CODE REVIEW

File: tendrl/gluster_integration/objects/definition/gluster.yaml [hunk 1]
```diff
@@ -105,7 +78,7 @@ namespace.gluster:
           help: raw data
           type: String
       enabled: true
-      value: clusters/$TendrlContext.integration_id/raw_map/
+      value: clusters/$TendrlContext.integration_id/nodes/$NodeContext.node_id/raw_map/
```
Comment (shtripat) [inline tendrl/gluster_integration/objects/definition/gluster.yaml]: If I understand correctly this for maintaining objects's raw representation. right? If so why under node specific dir?
Can you explain the logic a bit?

Comment (r0h4n) [reply tendrl/gluster_integration/objects/definition/gluster.yaml]: Yes. raw map changes per node. Hence saving pee node

Comment (shtripat) [reply tendrl/gluster_integration/objects/definition/gluster.yaml]: ok

File: tendrl/gluster_integration/sds_sync/__init__.py [hunk 2]
```diff
@@ -178,17 +178,18 @@ def run(self):
                                     connected=peers['peer%s.connected' % index]
                                 )
                             try:
-                                stored_peer_status = NS._int.client.read(
-                                    "clusters/%s/Peers/%s/connected" % (
-                                        NS.tendrl_context.integration_id,
-                                        peers['peer%s.uuid' % index]
-                                    )
+                                stored_peer_status = etcd_utils.read(
```
Comment (shtripat) [inline tendrl/gluster_integration/sds_sync/__init__.py]: NS.tendrl.objects.ClusterPeer(...).load() ?

**Comment by GowthamShanmugam** [comment]
LGTM

File: tendrl/gluster_integration/sds_sync/__init__.py (hunk 2 above)
Comment (r0h4n) [reply tendrl/gluster_integration/sds_sync/__init__.py]: This block of code will be moved to  GlusterPeer.on_change function soon

**Pull request merged**